app/models/setting/provisioning.rb

```diff
@@ -11,7 +11,7 @@ def self.local_boot_labels
     end
   end
 
-  Setting::BLANK_ATTRS.push(*(default_global_labels + local_boot_labels))
+  Setting::BLANK_ATTRS.push(*(default_global_labels + local_boot_labels + default_pxe_item_global + default_pxe_item_local))
```

Comment:
Sometimes, it's good thing to actually run the code :)

```
undefined local variable or method `default_pxe_item_global' for #<Class:0x0055b4d607b418> (NameError)
```

Reply:
Nah, that's too slow. I let Jenkins to run the code for me :-)